server/eventlog.go

```diff
@@ -416,6 +416,11 @@ func getSecureBootState(attestEvents []attest.Event) (*pb.SecureBootState, error
 	}, nil
 }
 
+func isValidEventTag(event *pb.Event) bool {
+	// TODO: verify the event digest once https://lore.kernel.org/all/20240308085754.476197-7-ardb+git@google.com/T/ is patched.
```

Comment:
We can't do this since it's not compatible with any unpatched images. Either we need to check for both the buggy measurement AND the fixed measurement, or we need to do what we're doing now and skip the check. I still maintain the check is not security critical, so I'd just remove the TODO.

@jkl73

Reply:
sgtm

Reply:
Done.